authors/apps/articles/management/commands/populate_impression.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import (
+    BaseCommand,
+    CommandError
+)
+
+from authors.apps.articles.models import Impression
+
+
+class Command(BaseCommand):
+    help = "Populate the impressions table."
+
+    def handle(self, *args, **options):
+        impressions = [
+            {'Like': 'Person finds the article appeasing'},
+            {'Dislike': 'Person does not find the article to their liking'},
+            {'Favourite': "Person has found that they resonate with an article's words"}
+        ]
+        try:
+            Impression.objects.get(name='Like')
+            message = 'No need to populate Impression table.'
+        except Impression.DoesNotExist:
+            for impression in impressions:
```

Comment:
Please have
```python
for key in impression:
    Impression.objects.create(
        name=key,
        description=impression[key]
     ).save()
```
become
```python
for key, value in impression:
    Impression.objects.create(
        name=key,
        description=value
     ).save()          
```

The second one is more intuitive to read as it leverages the `for key, value in dictionary` Python idiom.